Corona-Warn-App/src/main/res/layout/covid_certificate_validation_result_fragments.xml

```diff
@@ -16,15 +16,15 @@
             android:id="@+id/collapsing_toolbar_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:nestedScrollingEnabled="true"
             android:background="@drawable/covid_certificate_validation_gradient"
+            android:nestedScrollingEnabled="true"
             app:layout_scrollFlags="scroll|exitUntilCollapsed|snap"
             app:titleEnabled="false">
 
             <ImageView
                 android:id="@+id/header_image"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
+                android:layout_height="248dp"
```

Comment:
does this fit all screen sizes ?

Reply:
Good point, tested it and it worked fine! But after some more testing I figured out that the height is not different on various screen sizes when using match_parent compared to dp directly - so I reverted the changes since they do not add any value